source/PyMaterialX/PyDocument.cpp

```diff
@@ -17,7 +17,7 @@ void bindPyDocument(py::module& mod)
     py::class_<mx::Document, mx::DocumentPtr, mx::Element>(mod, "Document")
         .def("initialize", &mx::Document::initialize)
         .def("copy", &mx::Document::copy)
-        .def("importLibrary", &mx::Document::importLibrary)
+        .def("importLibrary", &mx::Document::importLibrary, py::arg("library"), py::arg("copyOptions") = (mx::CopyOptions*) nullptr)
```

Comment:
Minor note: In Python bindings, we usually place the argument list on its own indented line.  (See the declaration of copyContentFrom below.)